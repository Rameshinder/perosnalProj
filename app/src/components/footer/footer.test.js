import React from 'react'
import { shallow } from 'enzyme'

import Footer from './index'

describe('Footer', () => {
  const app = shallow(<Footer />)

  test('renders properly', () => {
    expect(app).toMatchSnapshot()
  })

  it('Footer should render without crashing', () => {
    shallow(<Footer />)
  })
})
