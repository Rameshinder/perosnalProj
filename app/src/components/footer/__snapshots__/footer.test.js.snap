// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <Styled(glamorous.Div)
        data-test-id="Footer"
      >
        <Styled(glamorous.Div)
          data-test-id="Text_Footer"
        >
          <span>
            © 
          </span>
          Australia and New Zealed Banking Group Limited (ANZ) 
          <span>
            2018 
          </span>
          <span>
             ABN 
          </span>
          <span>
            11 
          </span>
          <span>
            005 357 522.
          </span>
        </Styled(glamorous.Div)>
      </Styled(glamorous.Div)>,
      "data-test-id": "Container_Footer",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <Styled(glamorous.Div)
          data-test-id="Text_Footer"
        >
          <span>
            © 
          </span>
          Australia and New Zealed Banking Group Limited (ANZ) 
          <span>
            2018 
          </span>
          <span>
             ABN 
          </span>
          <span>
            11 
          </span>
          <span>
            005 357 522.
          </span>
        </Styled(glamorous.Div)>,
        "data-test-id": "Footer",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <span>
              © 
            </span>,
            "Australia and New Zealed Banking Group Limited (ANZ) ",
            <span>
              2018 
            </span>,
            <span>
               ABN 
            </span>,
            <span>
              11 
            </span>,
            <span>
              005 357 522.
            </span>,
          ],
          "data-test-id": "Text_Footer",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "© ",
            },
            "ref": null,
            "rendered": "© ",
            "type": "span",
          },
          "Australia and New Zealed Banking Group Limited (ANZ) ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "2018 ",
            },
            "ref": null,
            "rendered": "2018 ",
            "type": "span",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": " ABN ",
            },
            "ref": null,
            "rendered": " ABN ",
            "type": "span",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "11 ",
            },
            "ref": null,
            "rendered": "11 ",
            "type": "span",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "005 357 522.",
            },
            "ref": null,
            "rendered": "005 357 522.",
            "type": "span",
          },
        ],
        "type": [Function],
      },
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <Styled(glamorous.Div)
          data-test-id="Footer"
        >
          <Styled(glamorous.Div)
            data-test-id="Text_Footer"
          >
            <span>
              © 
            </span>
            Australia and New Zealed Banking Group Limited (ANZ) 
            <span>
              2018 
            </span>
            <span>
               ABN 
            </span>
            <span>
              11 
            </span>
            <span>
              005 357 522.
            </span>
          </Styled(glamorous.Div)>
        </Styled(glamorous.Div)>,
        "data-test-id": "Container_Footer",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <Styled(glamorous.Div)
            data-test-id="Text_Footer"
          >
            <span>
              © 
            </span>
            Australia and New Zealed Banking Group Limited (ANZ) 
            <span>
              2018 
            </span>
            <span>
               ABN 
            </span>
            <span>
              11 
            </span>
            <span>
              005 357 522.
            </span>
          </Styled(glamorous.Div)>,
          "data-test-id": "Footer",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <span>
                © 
              </span>,
              "Australia and New Zealed Banking Group Limited (ANZ) ",
              <span>
                2018 
              </span>,
              <span>
                 ABN 
              </span>,
              <span>
                11 
              </span>,
              <span>
                005 357 522.
              </span>,
            ],
            "data-test-id": "Text_Footer",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "© ",
              },
              "ref": null,
              "rendered": "© ",
              "type": "span",
            },
            "Australia and New Zealed Banking Group Limited (ANZ) ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "2018 ",
              },
              "ref": null,
              "rendered": "2018 ",
              "type": "span",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": " ABN ",
              },
              "ref": null,
              "rendered": " ABN ",
              "type": "span",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "11 ",
              },
              "ref": null,
              "rendered": "11 ",
              "type": "span",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "005 357 522.",
              },
              "ref": null,
              "rendered": "005 357 522.",
              "type": "span",
            },
          ],
          "type": [Function],
        },
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
